test/GithubApi.Redirect.js

```diff
@@ -0,0 +1,35 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const statusCode = require('http-status-codes');
+const { expect } = require('chai');
+
+describe('Given an old repo that changed name', () => {
+  const oldUrl = 'https://github.com/aperdomob/redirect-test';
+  const newUrl = 'https://github.com/aperdomob/new-redirect-test';
+
+  describe('When we consult the repo via HEAD services', () => {
+    let headQueryResponse;
+    before(() => agent.head(oldUrl)
+      .catch((err) => {
+        headQueryResponse = err;
+      }));
+
+    it('We verify that the repo did change, and get the new url', () => {
```

Comment:
be more specific using BDD, like `then the response should show that the repo has a new url`